src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,10 @@ Do you want to proceed?</value>
   <data name="TheRenameTrackingSessionWasCancelledAndIsNoLongerAvailable" xml:space="preserve">
     <value>The rename tracking session was cancelled and is no longer available.</value>
   </data>
+  <data name="Rename2" xml:space="preserve">
+    <value>Rename '{0}'</value>
+  </data>
+  <data name="SingleQuote" xml:space="preserve">
```

Comment:
If you look at the way these strings are used, there's actually three TextBlocks in a horizontal StackPanel, so those will be laid out Left-to-Right by WPF regardless of text direction. So for my comment, "Left" means on the left side visually, and "Right" means on the right side visually. 

So, an RTL language could place RTL text on the left (visually) and right (visually) such that the whole thing ends up looking correct when laid out. If we switch to manually splitting a single resource string with a {0} in the middle, we'd have to look for text direction markers in the resource before deciding which half of the split string goes into which TextBlock, and that feels even worse to me.

This is all happening because we're trying to use WPF to truncate the middle part of a larger string. As @rchande  and I were discussing yesterday, if we really wanted to we could get down to just one resource string by asking WPF to measure what the text size would be if we put in the full identifier, or various prefixes of it with trailing "..."s until we find an acceptable prefix, and then just use that. (Of course, in that case you'd have to watch out for text direction markers in the identifier and make sure the text on the other side of the {0} continues using the text direction from before the identifier was inserted.)

Or we could just ditch the quotes completely, or put the entire string in one TextBlock and let WPF truncate the whole thing (removing the closing single quote if necessary), etc. But, I'd rather not have to change Ravi's design due to loc being somewhat more difficult (as long as it's still solvable).